.github/workflows/python-publish.yml

```diff
@@ -0,0 +1,32 @@
+# This workflows will upload a Python Package using Twine when a release is created
```

Comment:
With Poetry, publishing to PyPI is `poetry publish --build`. But honestly I'd just leave out the publish workflow for now.

Reply:
Fixed in ac45042